mitosheet/src/mito/components/modals/StreamlitSignupModal.tsx

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Mito
+
+import React, { FormEvent, useState } from 'react';
+import { MitoAPI } from '../../api/api';
+import BlueMitoFolk from '../icons/mitofolks/BlueMitoFolk';
+import PinkMitoFolk from '../icons/mitofolks/PinkMitoFolk';
+import YellowMitoFolk from '../icons/mitofolks/YellowMitoFolk';
+import { ModalEnum } from './modals';
+
+import '../../../../css/signup-modal.css';
+import { AnalysisData, SheetData, UIState } from '../../types';
+import { classNames } from '../../utils/classNames';
+import Input from '../elements/Input';
+import TextButton from '../elements/TextButton';
+import { TaskpaneType } from '../taskpanes/taskpanes';
+
+const StepOne = (
+    props: {
+        next: () => void;
+        mitoAPI: MitoAPI;
+    }): JSX.Element => {
+
+    const [email, setEmail] = useState('');
+
+    
+    const onSubmit = async (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        await props.mitoAPI.updateSignUp(email);
+        props.next();
+    }
+
+    return (
+        <div className='signup-modal-left-column'>
+            <div>
+                <h1 className='text-header-1' style={{margin: 0}}>
+                    Sign up for Mito
+                </h1>
+                <p className='text-header-2 text-color-medium-important' style={{marginTop: '10px'}}>
+                    We’ll send you periodic product updates and welcome any feedback.
+                </p>
+                <p className='text-header-2 text-color-medium-important' style={{marginTop: '10px'}}>
+                    This modal appears the first time you use Mito for Streamlit.
```

Comment:
My understanding is if the developer goes through this signup process locally, then they will also have to go through it when they deploy to something like Streamlit Cloud. Maybe we can have a link here to more information or explain that more here. 

I don't think this message makes it clear that the end users of the streamlit app are not going to have to go through this.